tests/wrap.py

```diff
@@ -34,25 +34,22 @@ def testMakeControl(self):
         config.foo = 2
         config.bar.append("baz")
         control = config.makeControl()
-        self.assert_(testLib.checkControl(control, config.foo, config.bar))
+        self.assert_(testLib.checkControl(control, config.foo, config.bar._list))
```

Comment:
I think we want pybind to convert all kinds of python sequences to `std::vector` before we accept it as The Way.  SWIG will do it if you code it right (though getting it right can be tricky).